app/assets/stylesheets/centers.scss

```diff
@@ -0,0 +1,69 @@
+// Place all the styles related to the centers controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link {
+  color: orange;
+}
+
+a:visited {
+  color: grey;
+}
+
+a:hover {
+  color: blue;
+  background-color: yellow;
+}
+
+a:active {
+  color: #0000FF;
+}
+
+thead {
+  font-family: sans-serif;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #f2f2f2
+}
+
+h2 {
+  font-family: sans-serif;
+  text-align: left;
+  font-size: 32px;
+  padding: 15px;
+}
+
+table, th, td {
+  border-collapse: collapse;
+  font-family: sans-serif;
+  border: 1px solid black;
+}
+
+
+th { 
+  padding: 15px;
+  text-align: left;
+}
+
+td {
+  width: 440px;
+  text-align: left;
+}
+
+#center_name {
+  padding: 15px;
+}
+
+#details {
+  width: 60px;
+  text-align: center;
+}
+
+#data {
```

Comment:
instead of doing this, you can do `table#my-table td` to target the table's td(s)